security/operator/secopspolicy/destroy.yaml

```diff
@@ -1,8 +1,13 @@
+# This collection of tasks will lookup an Aporeto
+# `networkaccesspolicies` policy by filtering for a
+# known tag.
+# Variables
+#   - apo_namespace
 - hosts: localhost
   gather_facts: no
   vars:
     k8s_message_key: _secops_pathfinder_gov_bc_ca_networksecuritypolicy
-    namespace: /bcgov-devex/lab
+    apo_base_namespace: /bcgov/test
```

Comment:
Needs to be changed to       `  apo_namespace: "{{ lookup('env', 'APOCTL_BASE_NAMESPACE') }}/{{ metadata.namespace }}"`

Reply:
Fixed.